README.md

```diff
@@ -83,6 +83,7 @@ We accept but don't fully support the following types of images:
 Our `export` method provides support for the following options:
 
 - options `Object`
+    - format `String` - `png` or `jpeg`
```

Comment:
Can we update the phrasing have more explanation? People might not understand this outright. Maybe we can use the content from:

https://github.com/twolfson/phantomjssmith/tree/1.0.0#canvasexportoptions